bin/bootstrap-account.sh

```diff
@@ -2,8 +2,14 @@
 set -euxo pipefail
 
 # PROJECT_NAME defaults to name of the current directory.
+# Run this at project root before changing directories
 PROJECT_NAME=$(basename $(PWD))
 
+# GITHUB_REPOSITORY defaults to the origin of the current git repo
+# Get the URL string and remove the "git@github.com:" prefix, leaving
+# just the "org/repo" string (e.g. "navapbc/template-infra")
+GITHUB_REPOSITORY=$(git remote get-url origin | sed s/^git@github.com://)
```

Comment:
does this always work? my remote is formatted as `https://github.com/EOLWD/pfml.git` 🤔 (don't remember how I set it up, but I thought I used the SSH origin.)

Reply:
i updated the way we calculate the github repo can you take a look to make sure it is understandable and also works with your setup? i tested a few different url strings so i think it works.

Reply:
actually i'll go ahead and merge but feel free to let me know if you discover something afterwards